src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -631,6 +631,26 @@ extern "C" {
  */
 #define SIGNALING_CONNECT_TIMEOUT (5 * HUNDREDS_OF_NANOS_IN_A_SECOND)
 
+/**
+ * Default minimum number of threads in the threadpool for the SDK
+ */
+#define THREADPOOL_MIN_THREADS 5
```

Comment:
This feels high. It's 5 threads idling at all times.

Reply:
Do we want to make it 3? Ok with that. Considering there will peer connection and signaling, though 5 would be ok.